go.mod

```diff
@@ -18,7 +18,7 @@ require (
 	github.com/Mellanox/rdmamap v0.0.0-20191106181932-7c3c4763a6ee
 	github.com/Microsoft/ApplicationInsights-Go v0.4.2
 	github.com/Microsoft/go-winio v0.4.9 // indirect
-	github.com/Shopify/sarama v1.27.2
+	github.com/Shopify/sarama v1.29.0
```

Comment:
just to clarify you're not only patching sarama but also upgrading to the latest code in the process? Its not any issue but just curious if thats whats happening here.

Reply:
Yep, I updated to latest and then the line at the bottom of the file replaces this dep with the 128 fork. Technically could have stayed the same, but I think this is the right way to do it